src/core/virtual-routes/exact-match.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Adds beginning of input (^) and end of input ($) assertions if needed into a regex string
+ * @param {string} matcher the string to match
+ * @returns {string}
+ */
+export function makeExactMatcher(matcher) {
+  const matcherWithBeginningOfInput = matcher.startsWith('^')
+    ? matcher
+    : `^${matcher}`;
+
+  const matcherWithBeginningAndEndOfInput =
+    matcherWithBeginningOfInput.endsWith('$')
```

Comment:
Method `endsWith()` not supported by legacy browsers.

Reply:
took care of this. btw, how do you check for this? do you have a tool \ lint rule for that?

Reply:
Nope. Wish I did. 

I use a free [BrowserStack](https://www.browserstack.com) account to manually test in IE11. Browserstack has a nice [open source program](https://www.browserstack.com/open-source) that allows open source authors to use a limited set of their tools for free. Super handy.

Reply:
FYI, I noticed we have an `endsWith()` function here:

https://github.com/docsifyjs/docsify/blob/b8b221f3a87652acf6d57286004d711a3671537f/src/core/router/util.js#L117-L119

Not saying you have to use that, but you could add a `startsWith()` function in this same file and leverage those if you preferred.